tv_apps/smart-home/js/home.js

```diff
@@ -113,9 +114,35 @@
           // appearance is the same as it.
           that.searchButton.classList.add('hidden');
         }.bind(that));
+
+        document.addEventListener('visibilitychange', function() {
+          that.onVisibilityChange();
+        });
+        if (document.visibilityState === 'visible') {
+          that.onVisibilityChange();
+        }
       });
     },
 
+    onVisibilityChange: function() {
+      if (document.visibilityState === 'visible') {
+        var relation = {
+          child: {
+            child: true
+          }
+        };
```

Comment:
The JSON structure is not self-describing. Please consider one of them:
1. use querySelector.
2. let doBubbleAnimation accept only Scrollable and we wouldn't need 'relation' variable.
3. Comment clearly why you are using child: child: true.

I'd prefer 1 or 2.